src/scss/pages/_phonebook.scss

```diff
@@ -1,7 +1,15 @@
 .staff-list {
-  columns: auto 3;
+  columns: auto 1;
   column-rule: 1px solid $gray;
 
+  @include media-breakpoint-up(sm) {
```

Comment:
Consider using a standard grid or even a table here instead of CSS columns to prevent unpredictable expanding/collapsing of content.  I'm noticing some content tends to expand to a new column when we wouldn't want it to:

![screen shot 2017-11-07 at 10 46 45 am](https://user-images.githubusercontent.com/1649216/32506122-63ab16d4-c3a9-11e7-8e76-e5a5c483a70d.png)